tee-worker/service/src/prometheus_metrics.rs

```diff
@@ -55,6 +60,12 @@ lazy_static! {
 	static ref ENCLAVE_SIDECHAIN_TOP_POOL_SIZE: IntGauge =
 		register_int_gauge!("litentry_worker_enclave_sidechain_top_pool_size", "Enclave sidechain top pool size")
 			.unwrap();
+	static ref ENCLAVE_STF_CALLS: IntGaugeVec =
```

Comment:
I think you still need to rename calls -> tasks in few places, for example here in variable name and in arguments passsed to `register_int_gauge_vec`